recipe/meta.yaml

```diff
@@ -93,7 +93,7 @@ test:
     - conda-debug --help
 
   downstreams:
-    - conda-smithy  # [py>=311]
+    - conda-smithy  # [py>=311 and not osx]
```

Comment:
this can be reverted once the next version of `conda-smithy` is release (containing https://github.com/conda-forge/conda-smithy/commit/f30e378eab7c476eedb1f31755d03b03254ea16b)